steps/synopsys-pt-timing-signoff/pt.tcl

```diff
@@ -9,22 +9,23 @@ set report_default_significant_digits 3
 ##################################################################
 #    Constraint Analysis Section
 ##################################################################
-check_constraints -verbose > check_constraints.report
+check_constraints -verbose > reports/check_constraints.report
 
 ##################################################################
 #    Update_timing and check_timing Section                      #
 ##################################################################
 
 update_timing -full
-check_timing -verbose > check_timing.report
+check_timing -verbose > reports/check_timing.report
 
 ##################################################################
 #    Report_timing Section                                       #
 ##################################################################
-report_global_timing > report_global_timing.report
-report_clock -skew -attribute > report_clock.report 
-report_analysis_coverage > report_analysis_coverage.report
-report_timing -crosstalk_delta -slack_lesser_than 1000.0 -max_paths 100 -pba_mode exhaustive -delay min_max -nosplit -input -net > report_timing_pba.report
+report_global_timing > reports/report_global_timing.report
+report_clock -skew -attribute > reports/report_clock.report 
+report_analysis_coverage > reports/report_analysis_coverage.report
+report_timing -crosstalk_delta -slack_lesser_than 1000.0 -max_paths 100 -pba_mode exhaustive -delay max -nosplit -input -net -transition_time -path_type full_clock_expanded > reports/report_timing_setup_pba.report
+report_timing -crosstalk_delta -slack_lesser_than 1000.0 -max_paths 100 -pba_mode exhaustive -delay min -nosplit -input -net -transition_time -path_type full_clock_expanded > reports/report_timing_hold_pba.report
 
```

Comment:
Can we also name the reports similarly to the DC node?

- like .timing.setup.rpt
- and .timing.hold.rpt